src/util/ndjsonParsers.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Function to parse ndjson data into an array of javascript objects
+ */
+function parseNdjson(ndjsonString: string): unknown[] {
+  const jsonObjectArray = ndjsonString.split('\n').map(data => JSON.parse(data));
+
+  return jsonObjectArray;
+}
+
+/*
+ * Function to stringify an array of javascript objects into a string of new line delimitated
+ * json strings. Default has numValues set to 1 to just json.stringify one element of the passed in array
+ */
+function stringifyJsonArray(jsonObjectArray: unknown[], start: number, numValues: number = 1) {
```

Comment:
No its always 1. I left it in when I was testing displaying multiple resources per page. I will get rid of it because we probably won't ever need to display more than 1 resource per page in the modal.